server/app/services/question/PrimaryApplicantInfoTag.java

```diff
@@ -0,0 +1,105 @@
+package services.question;
+
+import com.google.common.collect.ImmutableList;
+import models.QuestionTag;
+import services.question.types.QuestionType;
+
+/**
+ * Defines tags that can be applied to a question to denote the answer is a primary piece of
+ * information about the applicant. These are 1:1 with particular QuestionTags.
+ */
+public enum PrimaryApplicantInfoTag {
+  APPLICANT_DOB(
+      QuestionTag.PRIMARY_APPLICANT_DOB,
+      QuestionType.DATE,
+      "primaryApplicantInfoDob",
+      "Applicant Date Of Birth",
+      "Setting this property will allow the answer to be pre-populated with the"
+          + " applicant's date of birth if a TI created this applicant."),
+  APPLICANT_EMAIL(
+      QuestionTag.PRIMARY_APPLICANT_EMAIL,
+      QuestionType.EMAIL,
+      "primaryApplicantInfoEmail",
+      "Applicant Email Address",
+      "Setting this property will allow the email address collected from this"
+          + " question to be used to email status updates to guest applicants, as well as make the"
+          + " application searchable by this address."),
+  APPLICANT_NAME(
+      QuestionTag.PRIMARY_APPLICANT_NAME,
+      QuestionType.NAME,
+      "primaryApplicantInfoName",
+      "Applicant Name",
+      "Setting this property will allow CiviForm to use the name to identify the"
+          + " user and their application in the UI, as well as make the application searchable by"
+          + " name."),
+  APPLICANT_PHONE(
+      QuestionTag.PRIMARY_APPLICANT_PHONE,
+      QuestionType.PHONE,
+      "primaryApplicantInfoPhone",
+      "Applicant Phone Number",
+      "Setting this property will make the application searchable by phone" + " number.");
+
+  private final QuestionTag tag;
+  private final QuestionType type;
+  private final String fieldName;
+  private final String displayName;
+  private final String description;
+
+  /**
+   * This contains information about a tag that can be applied to a question, marking it as a piece
```

Comment:
Suggestion (non-blocking): I got confused about what the different values passed into the enums above are and then saw this description when I scrolled down which explains it. What do you think about one of these options:
1. move the enum below this so this comment is the first thing in the file
2. expand the comment on the enum to include details about the params
3. add comments on the params in the enums to name them